repo2docker/buildpacks/conda/freeze.py

```diff
@@ -35,7 +41,7 @@
 yaml = YAML(typ="rt")
 
 
-def freeze(env_file, frozen_file):
+def freeze(env_file, frozen_file, containerruntime="docker", hostmountopts=""):
```

Comment:
Can we come up with a more descriptive name for `containerruntime`? When I started reading the code I was preparing myself for witnessing some compatibility magic dance but after reading it I think this is "just" the name of the executable to use. However `containerruntime_executable` is even worse than the current name, so I have no good alternative suggestion. But think it is worth word smithing for a few minutes to see if we can find a nice name.

Reply:
I'm not a big fan of the `containerruntime` name either, but I already went through a few other options before choosing this and I can't think of anything better. Anyone else have a suggestion?

Reply:
Maybe we stick with it and add a comment that is a sentence that says something like "This name is a bit weird. The variable contains the name of the container runtime executable."

Or we leave it as is :-/